app/components/Card/CardFront.tsx

```diff
@@ -48,12 +47,9 @@ export default function CardFront({
         >
           <Image
             src={image ?? placeholderImage}
-            fill={true}
-            style={{
-              objectFit: "cover",
-            }}
+            width={500}
+            height={500}
```

Comment:
I'm not sure why did you switch to sizing the image with width and height. Could you explain your reasoning? From what I understand, width and height properties determine aspect ratio that will be used for the image. So in this case, image's aspect ratio will be 1:1. When I compare how the image changes based on this, it looks like your change changes the aspect ratio and "zooms out" the image, which I actually like better.

Reply:
This was a bug I worked on for a long time where the image would disappear when the screen size was below 768. I tried everything and this was the only solution that worked. However, now that you pointed out that "base" is the needed first size in responsive settings (which I had not known about or tested), it works with the old logic, so I have now reverted it back to the way it was.

Reply:
Actually, I'm changing it back to how I had it. It looks good, and plus I remember now I was getting the following error without the sizes indicated:
<img width="408" alt="Screenshot 2024-04-29 at 11 48 47 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/76603041/f8b549bf-4dd0-4e61-a3b4-047f5697b129">